app/src/main/kotlin/com/ayatk/biblio/domain/usecase/LibraryUseCaseImpl.kt

```diff
@@ -30,23 +30,23 @@ import javax.inject.Singleton
 
 @Singleton
 class LibraryUseCaseImpl @Inject constructor(
-    private val novelRepository: NovelRepository,
-    private val indexRepository: IndexRepository,
-    private val schedulerProvider: SchedulerProvider
+  private val novelRepository: NovelRepository,
+  private val indexRepository: IndexRepository,
+  private val schedulerProvider: SchedulerProvider
 ) : LibraryUseCase {
 
   override val libraries: Flowable<List<Library>> =
-      novelRepository.savedNovels
-          .map { it.map { it.toLibrary() } }
-          .subscribeOn(schedulerProvider.io())
+    novelRepository.savedNovels
+      .map { it.map { it.toLibrary() } }
+      .subscribeOn(schedulerProvider.io())
 
   override fun update(): Completable =
-      Completable.fromCallable {
-        // TODO: 更新処理
-      }
-          .subscribeOn(schedulerProvider.io())
+    Completable.fromCallable {
+      // TODO: 更新処理
```

Comment:
**[detekt]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
This comment contains text that has been defined as forbidden in detekt.